### ISSUE TITLE
run setup depends on chalk

### ISSUE THREAD

**Issue by eiktyrner** [mxstbr/react-boilerplate#297]
Cloned repo branch v3.0.0
npm run setup

```
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/jacob/test_projects/react-boilerplate/internals/scripts/helpers/checkmark.js:2:13)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

solved by manually running 
`npm install chalk` 

Should it really depend on the user having chalk installed before initializing this awesome project? It seems a bit silly for coloring a checkmark in an init script to have that requirement imho.

**Comment by mxstbr** [comment]
Ooh, that was my bad, good catch! I added the checkmark thing to the setup script without realising that the dependencies aren't installed yet…

Would you mind submitting a PR to fix this? Thanks!

**Comment by mxstbr** [comment]
Fixed by @w01fgang in #303, thanks!
